### ISSUE TITLE
Unsubscribe from single subsocket?

### ISSUE THREAD

**Issue by thw0rted** [Atmosphere/atmosphere-javascript#262]
In [the docs](https://github.com/Atmosphere/atmosphere/wiki/atmosphere.js-API), the `unsubscribe` method is a top level function that, as far as I can tell, closes all known connections to all servers at once.  I don't see any method on `AtmosphereRequest` ("subsocket" in the example) that would allow me to stop the subscription for a single topic.  Is this not possible?

**Comment by thw0rted** [comment]
Thanks Ronny.  Is that method described anywhere in the docs?  [I don't see it](https://github.com/Atmosphere/atmosphere/search?q=unsubscribeUrl) but maybe I'm looking in the wrong place.

**Comment by ronnybremer** [comment]
To be honest, I am not sure. I was debugging an issue in our code where two sockets are used on the same page and discovered that call inside of the JS file. It seems to work and is public, so I hope its an official API...

**Comment by jfarcand** [comment]
Yes it is. The documentation needs to be updated....unfortunately I don't have time those days, but if you have the cycle, I can review and merge :-)
